VideoApp/VideoApp/Video/VideoAppCameraSource.swift

```diff
@@ -172,3 +167,25 @@ extension VideoAppCameraSource: CameraSourceDelegate {
         localMediaController?.cameraSourceWasInterrupted()
     }
 }
+
+private extension VideoCodec {
+    var targetSize: CMVideoDimensions {
+        switch self {
+        case .h264, .vp8:
+            // 640 x 480 squarish crop (1.13:1)
+            return CMVideoDimensions(width: 544, height: 480)
+        case .vp8Simulcast:
+            // 1024 x 768 squarish crop (1.25:1) on most iPhones. 1280 x 720 squarish crop (1.25:1) on the iPhone X
+            // and models that don't have 1024 x 768.
+            return CMVideoDimensions(width: 900, height: 720)
+        }
+    }
+    var cropRatio: CGFloat { CGFloat(targetSize.width) / CGFloat(targetSize.height) }
+    
+    func frameRate(isMultiparty: Bool) -> UInt {
```

Comment:
The frame rates here don't seem to match with the constants declared in `MultiPartyViewController.swift`

From what I am seeing, If it's a 1:1 call and not simulcast it should be `24`. If its a multiparty call and not simulcast, it should be `15`. If its simulcast it should always be 20.

Reply:
Yeah since I didn't really have comprehensive requirements I went with sort of a mashup in an effort to be surgical. So I kept the 20 and 24 values from video app and used the 15 value from MultiPartyExample since I thought multi party was the main thing we were trying to bring over. Not really sure what is desired but it's easy to change. What do you think?

Reply:
I'll take another look at MultiPartyViewController.

Reply:
I think I get it now. All of this configuration inside if statements and switches is so hard to follow especially after adding this PR. I want a config struct so bad.

Reply:
I think this is fixed now. Good catch.